src/Redux/rockets/action-makers.js

```diff
@@ -0,0 +1,7 @@
+import { getApi } from '../../components/rockets/rockets-api';
+import { rocketShowed } from './action-types';
+
+export const showRockets =() => async (dispatch) => {
+    const rockets = await getApi();
+     dispatch({ type: rocketShowed, payload: rockets });
+  };
```

Comment:
@basem909  I would like to suggest separating your actions into action creators, this will make them more reusable.

Reply:
this file is the action creator it's only the first action now the show rocket